cypress/e2e/tests/structureSurvey/noTrialSpecific/onboardPractice.cy.ts

```diff
@@ -0,0 +1,144 @@
+import moment from "moment";
+import "moment/min/locales";
+
+import {
+  clearCache,
+  loginWebAdmin,
+  logoutWebAdmin,
+  assignSurveyToPatient,
+  unassignSurveyToPatient,
+  calculatedTime,
+} from "../../../helpers/adminHelper/adminHelper";
+
+import {
+  verifySurveyCard,
+  ONBOARDING_PRACTICE_SURVEY_ID,
+  verifyOnboardingExperienceOptions,
+  verifyPleaseProvideFeedbackScreen,
+  verifyQuestionsNotBeenAnswered,
+  checkAlertMessage,
+  currentDate,
+  SCHEDULE_ONCE,
+  submitSurvey,
+  CONTAINED_PRIMARY_BUTTON,
+  ALERT_NOTIFICATION,
+  verifyInputMessageError,
+  OPTION_RADIO_EXCELLENT,
+  FEEDBACK_TEXT_INPUT,
+  QUESTIONS_TEXT_INPUT,
+  verifyOnboardPracticeConfirmation,
+} from "../../../helpers/surveyHelper";
+
+import {
+  loginWebPatient,
+  logoutWebPatient,
+} from "../../../helpers/loginHelper";
+
+import {
+  ONBOARDING_PRACTICE_SURVEY,
+  THIS_FIELD_CANNOT_BE_LEFT_BLANK,
+} from "../../../strings/surveyStrings";
+
+import { SURVEY_CARD_BUTTON } from "../../../helpers/homeHelper";
+import { NEXT } from "../../../helpers/generalStrings";
+
+describe("notTrialSpecific - Onboarding Practice Survey", () => {
```

Comment:
```suggestion
describe("Onboarding Practice Survey", () => {
```
Obs. the folder name should be `nonTrialSpecific`